### PR TITLE
Major: adding logs, exit code and a retry

### PR DESCRIPTION
**Pull request by mathild3r**

### CODE REVIEW

File: jira.go [hunk 1]
```diff
@@ -63,46 +63,59 @@ func getJiraTickets(endpointAPI string, orgID string, projectID string, token st
...
+	responseData := makeSnykAPIRequest("POST", endpointAPI+"/v1/org/"+orgID+"/project/"+projectInfo.K("id").String().Value+"/issue/"+vulnID+"/jira-issue", token, ticket)
+
+	return responseData
+}
+
+func openJiraTickets(endpointAPI string, orgID string, token string, jiraProjectID string, jiraTicketType string, assigneeID string, labels string, projectInfo jsn.Json, vulnsForJira map[string]interface{}, priorityIsSeverity bool) string {
+	fullResponseDataAggregated := ""
+
+	for _, vulnForJira := range vulnsForJira {
+		responseDataAggregatedByte := openJiraTicket(endpointAPI, orgID, token, jiraProjectID, jiraTicketType, assigneeID, labels, projectInfo, vulnForJira, priorityIsSeverity)
```
Comment (aarlaud) [inline jira.go:111]: probably cleaner to have openJiraTicket return responseDataAggregatedByte, err := and use if err != nil then retry opening a ticket again. Go has this nice multi member return value thingy that is nice to use for this sort of things. It's cleaner than parsing the response. And if parsing the response is the only way, then keep it into openjiraticket logic so we contain the opening logic inside the function.

File: jira.go [hunk 2]
```diff
@@ -63,46 +63,59 @@ func getJiraTickets(endpointAPI string, orgID string, projectID string, token st
...
+
+		if strings.Contains(string(responseDataAggregatedByte), "Field 'priority'") && priorityIsSeverity == true {
+			fmt.Println("Retrying with priorityIsSeverity set to false, max retry 1")
+			responseDataAggregatedByte = openJiraTicket(endpointAPI, orgID, token, jiraProjectID, jiraTicketType, assigneeID, labels, projectInfo, vulnForJira, false)
```
Comment (aarlaud) [inline jira.go:115]: how do we know we don't end up retrying again and again? if it fails for some other reasons? maybe "Highest" is not the default value after all. Probably should have a retry counter or something as failsafe

File: jira.go (hunk 1 above)
Comment (mathild3r) [reply jira.go]: I pushed some changes. But I want to keep openJiraTicket and openJiraTickets separated as I need to call openJiraTicket twice.

Comment (aarlaud) [reply jira.go]: Yes it's fine, the important bit is that openjiraticket handles the ticket opening and deals with the error interpretation. That needs to stay inside this function, so the change is good

**Pull request merged**